config/vxlan.py

```diff
@@ -56,6 +60,12 @@ def del_vxlan(db, vxlan_name):
     if(vxlan_count > 0):
         ctx.fail("Please delete all VLAN VNI mappings.")  
 
+    vnet_table = db.cfgdb.get_table('VNET')
+    vnet_keys = vnet_table.keys()
+    for vnet_key in vnet_keys:
+        if ('vxlan_tunnel' in vnet_table[vnet_key] and vnet_table[vnet_key]['vxlan_tunnel'] == vxlan_name):
```

Comment:
It might be worth including the `vnet_key` var in the error message to let the user know which vnets need to be removed?

Reply:
@ITJamie There might be multiple vnets pointing to a vxlan tunnel. In such case the  error message would be very verbose to the user. Additionally there are no commands to remove the VNET today in SONiC. The user has to modify the config_db.json and do reboot or config reload.
Lastly if we take other similar messages we don't print the references (e.g. Adding a portchannel member to a vlan will print "Ethernetx is part of portchannel!". 
Aligning with all existing error messages and the restrictions around VNET I believe the error message in current form should be sufficient